pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -140,6 +140,14 @@ func (l *LibvirtDomainManager) SyncVMI(vmi *v1.VirtualMachineInstance, useEmulat
 	// Set defaults which are not coming from the cluster
 	api.SetObjectDefaults_Domain(domain)
 
+	// Adjust guest vcpu config. Currenty will handle vCPUs to pCPUs pinning
```

Comment:
hm why do you not set VCPU pinning under `converter.go`?

Reply:
Yeah. Strictly speaking  the cpu discovery information should be placed in the converter context and the converter should do the conversion.

Reply:
> hm why do you not set VCPU pinning under converter.go?
I thought that all of the cpu discovery doesn't belong in the converter that basically should convert from api to domain xml, only setting the CPUTune belongs there, but it was too small. But you are right... it should be there.